src/ui/StepManager.js

```diff
@@ -118,6 +152,9 @@ StepManager.prototype.stepOverForward = function () {
     return
   }
   var step = this.traceManager.findStepOverForward(this.currentStepIndex)
```

Comment:
In solidity mode, step over forward should treat external and internal function calls in the same way.